pages/pokedex.tsx

```diff
@@ -21,54 +15,48 @@ function Pokedex() {
 
   const theme = `linear(to-l,#${firstValue},#${secondValue})`;
 
-  let offset = parseInt((router.query.offset as string) ?? "1");
-
-  const { isLoading, error, data } = useQuery(["content", offset], async () => {
-    const response = await axios.get(
+  const fetch = ({ pageParam: offset = 1 }) => {
+    return axios.get(
       `http://localhost:8081/api/pokemon?offset=${offset}&pageSize=24`
     );
-    const data = await response.data;
-    return data;
-  });
+  };
```

Comment:
Nothing wrong here just a cool enhancement to know about. Since we're returning in the same line as the function being created we can use the return syntax of an arrow function (I call these closures) `() => {}`.
The right-hand side of the arrow is what we are returning (when we don't return it has the return type of `void`), so we can remove the `{}`. 
```tsx
const fetch = ({ pageParam: offset = 1 }) =>
  axios.get(`http://localhost:8081/api/pokemon?offset=${offset}&pageSize=24`);
```
Note, when implicitly returning an object in the same fashion, you'll need to wrap the object you're returning in `({})` to let javascript know we mean 'return this object', instead of 'run this function'.
```tsx
const newPropAdder = (someObj) => ({...someObj, newProp: 'value' })
```